test/lib/parse/url.js

```diff
@@ -20,8 +20,9 @@ describe('isValidUrl', () => {
     context('when the url ends in /by', () => {
```

Comment:
i would revert the change here, and add another context like `when the url ends in abby` 😆

Reply:
and done